src/components/books/categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Category() {
+  return <div className="cat">Under Construction</div>;
```

Comment:
- [ ] as part of the requirements, the Category component should have a `<Button>` with the text "Check status" , kindly fix it 🙏